packages/cli/src/config/config.ts

```diff
@@ -229,7 +229,6 @@ export async function parseArguments(settings: Settings): Promise<CliArgs> {
         .option('screen-reader', {
           type: 'boolean',
           description: 'Enable screen reader mode for accessibility.',
-          default: false,
```

Comment:
Does this mean the code is not handling default values properly? Instead of this, maybe we should be checking how these values are handled and handle default values properly.

Reply:
Based on our offline discussion, this actually should be ok, since these are cmd line options and they take precedence over settings. Specifying a default value here will look similar to a user input. So we should not be doing that.

As suggested, please also verify other args that have default values like checkpointing, showMemoryUsage, etc and fix those if required.